pnc-ui/Gruntfile.js

```diff
@@ -415,6 +415,7 @@ module.exports = function (grunt) {
             '!index.html',
             'images/{,*/}*.{webp}',
             'fonts/{,*/}*.*',
+            'keycloak.json'
```

Comment:
So the keycloak file is also added to pnc-web? Do we need to secure frontend also?